tests/test_display.py

```diff
@@ -23,24 +40,81 @@ def test_symbols():
     # This test has no assertions, except from the human eye. When it fails,
     # ticks and crosses fail and must be inspected by the human eye
     if console.has_unicode():
-        symbols = display.unicode_symbols
+        symbol_groups = (display.html_symbols, display.unicode_symbols, display.ascii_symbols)
     else:
-        symbols = display.ascii_symbols
-    print("[{0}][{1}][{2}][{3}]".format(
-        symbols[py_trees.common.Status.SUCCESS],
-        symbols[py_trees.common.Status.FAILURE],
-        symbols[py_trees.common.Status.INVALID],
-        symbols[py_trees.common.Status.RUNNING]
+        symbol_groups = (display.html_symbols, display.ascii_symbols)
+
+    for symbols in symbol_groups:
+
+        print("Status: [{0}][{1}][{2}][{3}]".format(
+            symbols[py_trees.common.Status.SUCCESS],
+            symbols[py_trees.common.Status.FAILURE],
+            symbols[py_trees.common.Status.INVALID],
+            symbols[py_trees.common.Status.RUNNING]
+            )
         )
-    )
 
-    print("[{0}][{1}][{2}][{3}]".format(
-        symbols[py_trees.behaviours.Behaviour],
-        symbols[py_trees.composites.Sequence],
-        symbols[py_trees.composites.Selector],
-        symbols[py_trees.composites.Parallel]
+        print("Classes: [{0}][{1}][{2}][{3}]".format(
+            symbols[py_trees.behaviours.Behaviour],
+            symbols[py_trees.composites.Sequence],
+            symbols[py_trees.composites.Selector],
+            symbols[py_trees.composites.Parallel]
+            )
         )
+
+
+def test_xhtml_tree():
+    console.banner("Ascii/Html Snapshots - Comparison Check")
+
+    root = py_trees.composites.Selector("Selector")
+    high_priority = py_trees.behaviours.Count(
+        name="High Priority",
+        fail_until=1,
+        running_until=1,
+        success_until=10
     )
+    low_priority = py_trees.behaviours.Running(name="Low Priority")
+    root.add_children([high_priority, low_priority])
+    tree = py_trees.trees.BehaviourTree(root)
+    snapshot_visitor = py_trees.visitors.SnapshotVisitor()
+    tree.visitors.append(snapshot_visitor)
+    tree.tick()
+    tree.tick()
+    snippets = {}
+    # Visited
+    snippets["visited_ascii_tree"] = py_trees.display.ascii_tree(
+            tree.root,
+            visited=snapshot_visitor.visited,
```

Comment:
BTW test using show_status=True as well would be nice.

Reply:
Added